### PR TITLE
[entropy_src/doc] moving to D2

### PR DESCRIPTION
**Pull request by mwbranstad**
All major features and functions have been integrated and documented for achieving D2 for the entropy_src block.

Signed-off-by: Mark Branstad <mark.branstad@wdc.com>

### CODE REVIEW

File: hw/ip/entropy_src/doc/checklist.md
Comment (imphil) [inline hw/ip/entropy_src/doc/checklist.md:43]: @moidx and/or @cdgori  Can you confirm that the entropy source has all necessary features implemented?

**Review by imphil — APPROVED** [review]
LGTM, but let's please wait for feedback to confirm that all necessary functionality is indeed implemented.

**Comment by mwbranstad** [comment]
@moidx @cdgori I did not hear any objections on closing this PR because of missing function in today's meeting. Please comment now if that is not the case. I will merge this tomorrow if I do not hear anything. This does NOT include changes due to security hardening - that will be an ongoing set of issues that will be addressed separately.

File: hw/ip/entropy_src/doc/checklist.md
Comment (moidx) [reply hw/ip/entropy_src/doc/checklist.md:43]: health tests and boot and runtime init flows meet expected requirements. 
 
Note on software access:  `FW_OV_CONTROL` will be used to access raw entropy in software and to perform a KAT test of the conditioner. `ENTROPY_DATA` output will still be required to check the KAT output of the conditioner. 

Security hardening will be covered as part of D3 review. Security review is WIP.

**Review by moidx — APPROVED** [review]

**Pull request merged**
